shell/common/engine.h

```diff
@@ -234,6 +236,9 @@ class Engine final : public RuntimeDelegate {
   ///                                tasks that require access to components
   ///                                that cannot be safely accessed by the
   ///                                engine. This is the shell.
+  /// @param      platform_view      The object used by engine to create the
```

Comment:
This param is now out of date. Please also make sure to specify the updated threading restrictions when you fix this.

Reply:
Done.

Reply:
Done.